osidb/filters.py

```diff
@@ -170,6 +172,31 @@ def include_fields_filter(self, queryset, name, value):
         )
 
 
+class ExcludeFieldsFilterSet(SparseFieldsFilterSet):
```

Comment:
This class seems to be unused (see `FlawFilter(DistinctFilterSet, IncludeFieldsFilterSet)` and similarly)

Reply:
Good catch, thanks!

Reply:
If everything that inherits from `IncludeFieldsFilterSet` will also inherit from  `ExcludeFieldsFilterSet`, could we join these two together? Or would it be a bad practice?

Reply:
It wouldn't be bad practice, in fact we could probably invert the dependency to just have every class just use `SparseFieldsFilterSet`, but having them separate gives us the flexibility of having new views with either/or, exclude_fields is less useful than include_fields and less performant in general, so I can see a future in which new views only use `include_fields`